mltsp/tests/test_flask_app.py

```diff
@@ -1554,7 +1554,7 @@ def test_prediction_proc(self):
             entry = r.table("predictions").get("TEMP_TEST01").run(conn)
             pred_results_list_dict = entry
             assert(pred_results_list_dict["pred_results_list_dict"]
-                   ["TESTRUN_215153.dat"][0][0] in ['Beta_Lyrae',
+                   ["TESTRUN_215153"][0][0] in ['Beta_Lyrae',
                                                     'Herbig_AEBE'])
```

Comment:
It may be worth installing a pyflakes checker for your editor (it
typically finds this sort of thing quite easily).